awx/api/generics.py

```diff
@@ -88,15 +88,30 @@ def get(self, request, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         ret = super(LoggedLoginView, self).post(request, *args, **kwargs)
+
+        client_addr = request.META.get('REMOTE_ADDR', None)
+
+        # If an authorized header contains an IP, override the client_addr
+        for custom_header in settings.REMOTE_HOST_HEADERS:
```

Comment:
I'm not familiar with this code base, but I don't see how this loop will only apply to IP addresses. Won't this loop through every REMOTE_HOST_HEADERS? Like if it were `['REMOTE_ADDR', 'REMOTE_HOST']`, then would it not loop through those? The way the loop is constructed, it looks like it will overwrite `client_addr` with the first non-falsey value it finds in the list of `REMOTE_HOST_HEADERS`. Can you explain what is going on here and/or refactor this loop?